daprdocs/content/en/developing-applications/building-blocks/conversation/conversation-overview.md

```diff
@@ -0,0 +1,53 @@
+---
+type: docs
+title: "Conversation overview"
+linkTitle: "Overview"
+weight: 1000
+description: "Overview of the conversation API building block"
+---
+
+{{% alert title="Alpha" color="primary" %}}
+The conversation API is currently in [alpha]({{< ref "certification-lifecycle.md#certification-levels" >}}).
+{{% /alert %}}
+
+
+Using the Dapr conversation API, you can reduce the complexity of interacting with Large Language Models (LLMs) and enable critical performance and security functionality with features like prompt caching and personally identifiable information (PII) data obfuscation.
+
+## Features
+
+### Prompt caching
+
+todo
```

Comment:
```suggestion
Prompt caching stores frequently prompt in a cache to reuse this cached information instead of reprocessing it for every new request, which significantly reduces latency and cost. Essentially, it's a way to optimize performance by storing and reusing prompts that are often repeated across multiple API calls.
```